app/modules/asset_groups/models.py

```diff
@@ -1025,7 +1039,7 @@ def create_from_metadata(cls, metadata):
         with db.session.begin(subtransactions=True):
             db.session.add(asset_group)
 
-        log.info('created asset_group %r' % asset_group)
+        log.debug('created asset_group %r' % asset_group)
```

Comment:
The password printed out is the random password of the anonymous user created as part of this upload. 
But this does lead to another more serious security question:
Each request to EDM does print out the password. I suspect it shouldn't although this PR has not added this, only changed when it happens.